docs_nnx/nnx_basics.ipynb

```diff
@@ -293,22 +299,13 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Transforms\n",
+    "## Flax NNX transformations\n",
```

Comment:
Sounds good! We still mention "NNX" in blocks of text outside of titles and section names - in case some users need reassurance that it's the new API, not Linen.